index.d.ts

```diff
@@ -184,6 +215,7 @@ export declare namespace googleTranslateApi {
 		"cnh" = "Hakha Chin",
 		"ha" = "Hausa",
 		"haw" = "Hawaiian",
+		"he" = "Hebrew",
```

Comment:
From what I recall there was some reason languages shouldn't be included twice here, I'd like to keep that, if only so we encourage people to stick to a more consistent code

Reply:
I removed the line